src/Bundle/CoreBundle/Form/LinkType.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: stefankamsker
+ * Date: 17.09.18
+ * Time: 14:19
+ */
+
+namespace UniteCMS\CoreBundle\Form;
+
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class LinkType extends AbstractType implements DataTransformerInterface
+{
+
+    const URL_TARGETS = [
```

Comment:
All strings should come from translations, you should only use translation keys here. 
Maybe the options will get translated automatically by Symfony, otherwise you must do the translation here.